packages/waku/src/lib/middleware/rsc/ssr.ts

```diff
@@ -299,7 +299,7 @@ export const renderHtml = async <Context>(
                   file.startsWith('@fs/')
                     ? // FIXME This is ugly. We need to refactor it.
                       // remove '@fs'(3) on Unix and '@fs/'(4) on Windows
-                      file.slice(2 + path.sep === '/' ? 3 : 4)
+                      file.slice(2 + (path.sep === '/' ? 1 : 2))
```

Comment:
Is `2 +` meaningful? Otherwise:
```suggestion
                      file.slice(path.sep === '/' ? 3 : 4)
```